packages/flutter/lib/src/widgets/icon_theme_data.dart

```diff
@@ -26,36 +26,43 @@ class IconThemeData with Diagnosticable {
   ///
   /// The opacity applies to both explicit and default icon colors. The value
   /// is clamped between 0.0 and 1.0.
-  const IconThemeData({this.color, double opacity, this.size}) : _opacity = opacity;
+  const IconThemeData(
+      {this.color, double opacity, this.size, this.iconButtonSplashRadius})
+      : _opacity = opacity;
```

Comment:
I would check the [style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo) on this, it's a bit hard to read. :)